toolchain/semantics/semantics_ir.h

```diff
@@ -200,16 +200,17 @@ class SemanticsIR {
 
   // Adds an string, returning an ID to reference it.
   auto AddString(llvm::StringRef str) -> SemanticsStringId {
-    // If the string has already been stored, return the corresponding ID.
-    if (auto existing_id = GetStringID(str)) {
```

Comment:
Can GetStringID be removed now? I think it might've only been called here.

Reply:
It can! Thanks, done.